### PR TITLE
Auto disclosure QA and merge

### PR DESCRIPTION
**Pull request by adam-nygate**
Looks like we'll have to move towards hyphens in the url to fix the %2F issues

EDIT: this description is now deprecated, please ignore.

### CODE REVIEW

**Comment by JamieSlome** [comment]
@adam-nygate - awaiting the newest changes to `huntr-functions` and will look to clarify other work that needs to be completed for this change to not be breaking.

Thanks! 🍰

**Comment by JamieSlome** [comment]
Don't merge this in until I have had a read through, pls! 🍰

File: tools/process-bounties/indexer.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ bounties.withPromise().then(async bountyPaths => {
             { Bounty: { Credit: parseInt(vulnerabilityDetails.CVSS.Score * 100), Cash: 25 } }
 
         bountiesToIndex.push({
-            "ID": `${vulnerabilityDetails.PackageVulnerabilityID}-${vulnerabilityDetails.Package.Registry}-${encodeURIComponent(vulnerabilityDetails.Package.Name)}`,
+            "ID": `${vulnerabilityDetails.PackageVulnerabilityID}-${vulnerabilityDetails.Package.Registry}-${vulnerabilityDetails.Package.Name.replace('/','-')}`,
```
Comment (JamieSlome) [inline tools/process-bounties/indexer.js:38]: Revert this change for now, and we will track as an alternative PR.

**Review by adam-nygate — COMMENTED** [review]
Reviewed the diff; looks all ok, but will require us to accept the '/' to '-' for URLs when merging this in

File: tools/process-bounties/indexer.js (hunk 1 above)
Comment (adam-nygate) [reply tools/process-bounties/indexer.js:38]: Lol, was also just reviewing - will do.

Comment (JamieSlome) [reply tools/process-bounties/indexer.js:38]: Cheers! 🍰

**Review by adam-nygate — COMMENTED** [review]
Prettier kicked in, but the replacement of encodeURIComponent has been reverted.

**Comment by adam-nygate** [comment]
NOTE: Do not merge until ready to auto-merge disclosures

**Comment by JamieSlome** [comment]
@adam-nygate - can you label:

`WIP: [relevant PR title here]`?

Cheers 👍
